code/web/src/modules/survey/Survey/List.js

```diff
@@ -59,15 +61,31 @@ class List extends PureComponent {
               this.props.surveyItems.isLoading
                 ? <Loading/>
                 : this.props.surveyItems.surveyItems.length > 0
-                  ? this.props.surveyItems.surveyItems.map(item => (
-                    <div key={item.id} style={{ margin: '2em', float: 'left' }}>
+                  ? this.props.surveyItems.surveyItems.map((item, i) => (
+                    <div
+                      key={'item#'+i+'survey#'+this.props.match.params.page}
+                      style={{ margin: '2em', float: 'left' }}>
                       <Item item={item}/>
                     </div>
                   ))
                   : <EmptyMessage message="Something went wrong. Please go back and try again!" />
             }
           </GridCell>
         </Grid>
+
+        <Grid justifyCenter={true}>
+          <GridCell justifyCenter={true} style={{ padding: '2em', width: "50%"}}>
+            <Link to={surveyRoutes.survey.path( parseInt(this.props.match.params.page) - 1 )}>
+              <Button type="button" theme="secondary" disabled={this.props.match.params.page === '1'}>Back</Button>
+            </Link>
+          </GridCell>
+
+          <GridCell justifyRight={true} style={{ padding: '2em'}}>
+            <Link to={surveyRoutes.survey.path( parseInt(this.props.match.params.page) + 1 )}>
+              <Button type="button" theme="secondary" disabled={false}>Next</Button>
```

Comment:
What does the theme do?

Reply:
In `ui/Button.js` there's a `theme` prop referenced constantly. 'Secondary' is one of the pre-built options